veecle-telemetry-macros/test-crates/renamed-veecle-telemetry-crate/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+/// Test that the macro works when veecle-telemetry is renamed.
+
+// Synchronous functions
```

Comment:
```suggestion
// Synchronous functions:
```
Same for other comments.